app/views/recipes/modal.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= turbo_frame_tag "modal" do %>
+  <%= tag.div data: { controller: "turbo-modal" }, style: "padding: 1rem; border: 1px solid black" do %>
+    <h2>Modal</h2>
+    <p>Here goes the data of your modal</p>
+    <%= button_tag "Close", data: { action: "turbo-modal#hideModal" }, type: "button" %>
+  <% end %>
```

Comment:
[COMMENT, NOT REQUIRED YET] Be sure that once @kkolade branch has been merged you update your modal to have the requested functionality.